examples/foc/Kconfig

```diff
@@ -69,12 +69,6 @@ config EXAMPLES_FOC_NOTIFIER_FREQ
 	---help---
 		Select the FOC notifier frequency
 
-config EXAMPLES_FOC_IPHASE_ADC
```

Comment:
What about the warnings in `examples/foc/foc_motor_b16.c`?

Reply:
should be good now. It was rebased against a wrong branch